.readthedocs.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+
+build:
+  image: latest
+
+python:
+  version: 3.7
```

Comment:
AFAIR they don't ship 3.7 in their images currently.

Reply:
yes. now yes. maybe it was a crash in travis-ci
but python version 3.8.0a0 https://travis-ci.com/aio-libs/aiohttp/jobs/173498419
configuration of this build  https://travis-ci.com/aio-libs/aiohttp/jobs/173498419/config

the python3.7 version is stable, then it is worth pointing it out explicit. And build `nightly` is worth pointing with probability of error.

Reply:
@vir-mir this is not a Travis config, it's a RTD config